bin/init.bash

```diff
@@ -319,6 +319,9 @@ update_config() {
         log_info "Creating ${config_name} config"
     fi
 
+    sed 's/\./\$/g' "${override_config}" > tmp-config.yaml
```

Comment:
Should we specify somewhere that it's now not allowed to use `$` in keys in the config